cloudbuild-test.json

```diff
@@ -1,13 +1,9 @@
 {
   "steps": [
     {
-      "name": "gcr.io/dataform-public/dataform-builder@sha256:413b7e076ae77f1bed9b254434919e40d4d4112c00e1a12ea076da112a7012e9",
+      "name": "gcr.io/cloud-builders/bazel:5.4.0",
       "entrypoint": "bash",
-      "args": ["./scripts/cloudbuild/bazel_test"]
+      "args": ["./scripts/run_tests"]
     }
-  ],
-  "timeout": "3600s",
```

Comment:
Can you not remove these?
Beefy machine is definitely advantageous! Timeout though is probably a bit high based on my experience recently